src/pages/ProAmmPool/index.tsx

```diff
@@ -118,7 +117,7 @@ export default function ProAmmPool() {
 
   const debouncedSearchText = useDebounce(searchValueInQs.trim().toLowerCase(), 300)
 
-  const [showClosed, setShowClosed] = useState(true)
```

Comment:
intent or not

Reply:
intent

Reply:
user complains about it. then product changes requirement. LOL